com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -171,51 +169,6 @@ public class NetworkConfig
         /// Whether or not to enable network logs.
         /// </summary>
         public bool EnableNetworkLogs = true;
-        /// <summary>
```

Comment:
I think your encryption commit got mixed up here a little

Reply:
this PR branch is based on encryption removal branch. I'm planning to submit this PR after encryption removal PR goes in.